### PR TITLE
Fix eigen cholesky

### PR DESCRIPTION
**Pull request by roblim1**
This PR fixes issue #217.

### CODE REVIEW

**Comment by roblim1** [comment]
My [config_eigen.txt](https://github.com/verificarlo/verificarlo/files/6168844/config_eigen.txt), to illustrate that this fix enables function instrumentation with Verificarlo for the Cholesky benchmark.  All derived types aren't intercepted at the moment.  I have an implementation that will enable that and need to verify that all cases are covered.

File: src/libvfcfuncinstrument/libVFCFuncInstrument.cpp [hunk 1]
```diff
@@ -123,8 +123,9 @@ unsigned int getSizeOf(Value *V, const Function *F) {
   // of F
   for (auto &Args : F->args()) {
     if (&Args == V) {
-      for (const auto &U : F->users()) {
-        if (const CallInst *call = cast<CallInst>(U)) {
+      for (const auto &U : V->users()) {
+        if (isa<CallInst>(U)) {
```
Comment (pablooliveira) [inline src/libvfcfuncinstrument/libVFCFuncInstrument.cpp:127]: Nice catch @roblim1 !

**Review by pablooliveira — APPROVED** [review]

**Comment by pablooliveira** [comment]
Thanks for your contribution ! I am happy you are finding the tool useful for your purposes in eigen.

**Comment by roblim1** [comment]
Thanks @pablooliveira !  Much appreciated :-)

**Pull request merged**